src/green_recognition/include/GreenFilter.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * Created By: Robyn Castro
+ * Created On: June 15, 2017
+ * Description: Anything green is mapped to white, everything
+ *              else is mapped to black.
+ *
```

Comment:
LOL. Ok, so I made a bunch of comments on `CircleDetection.h` above, but I think that you may be using this file now, so just apply my comments from `CircleDetection.h` where they're appropriate.